src/main/java/dev/jbang/Script.java

```diff
@@ -183,6 +185,21 @@ public boolean enableCDS() {
 		return !collectRawOptions("CDS").isEmpty();
 	}
 
+	public String javaVersion() {
+		List<String> opts = collectRawOptions("JAVA");
+		if (!opts.isEmpty()) {
+			// If there are multiple //JAVA_VERSIONs we'll use the last one
```

Comment:
```suggestion
			// If there are multiple //JAVAs we'll use the last one
```

why not the first one ? thats the logic for others if I remember right?

Reply:
Is it? I didn't realize that. It just seemed more logical that way, but I don't mind one way or the other.